### PR TITLE
Fix minor typo

### PR DESCRIPTION
**Pull request by MattHeard**
---
Please include the following checklist in your PR:

* [x] Patches conform to the [coding conventions](https://github.com/haskell/cabal/blob/master/CONTRIBUTING.md#conventions).
* [x] Any changes that could be relevant to users have been recorded in the changelog (add file to `changelog.d` directory).
* [x] The documentation has been updated, if necessary.

Please also shortly describe how you tested your change. Bonus points for added tests!

### CODE REVIEW

File: Cabal/doc/getting-started.rst [hunk 1]
```diff
@@ -164,6 +164,6 @@ What Next?
 ----------
 
 Now that you know how to set up a simple Haskell package using Cabal, check out
-some of thee resources on the Haskell website's `documentation page
+some of these resources on the Haskell website's `documentation page
```
Comment (fgaz) [inline Cabal/doc/getting-started.rst:167]: I think it should be "the", otherwise it doesn't make much sense...

**Review by fgaz — CHANGES_REQUESTED** [review]

File: Cabal/doc/getting-started.rst (hunk 1 above)
Comment (MattHeard) [reply Cabal/doc/getting-started.rst:167]:
```suggestion
some of the resources on the Haskell website's `documentation page
```

**Comment by phadej** [comment]
Please squash commits. Two commits to fix one typo looks bad in history!

**Comment by MattHeard** [comment]
> Please squash commits. Two commits to fix one typo looks bad in history!

Both commits are now squashed into one.

**Comment by phadej** [comment]
My bad. I noticed just now that this PR is against 3.4 branch. Please don't make pull requests against release branches.  @fgaz or @MattHeard could you make a PR forward-porting this patch?

**Comment by fgaz** [comment]
Github puts the target branch in such a tiny font and hidden place that it's difficult to notice :-/

#7029

**Pull request merged**